### ISSUE TITLE
Unable to use redis with Klov

### ISSUE THREAD

**Issue by ssureshraja** [anshooarora/klov#29]
Hello @anshooarora ,
I'm running klov-0.0.3 in my machine and everything looks good until I enable redis. When redis is enabled, I get below exception while accessing "http://localhost:7779/builds"

```
org.springframework.data.redis.serializer.SerializationException: Cannot serialize; nested exception is org.springframework.core.serializer.support.SerializationFailedException: Failed to serialize object using DefaultSerializer; nested exception is java.lang.IllegalArgumentException: DefaultSerializer requires a Serializable payload but received an object of type [com.aventstack.klov.controllers.ReportViewController]
	at org.springframework.data.redis.serializer.JdkSerializationRedisSerializer.serialize(JdkSerializationRedisSerializer.java:92) ~[spring-data-redis-1.7.5.RELEASE.jar:na]
	at org.springframework.data.redis.core.AbstractOperations.rawHashValue(AbstractOperations.java:168) ~[spring-data-redis-1.7.5.RELEASE.jar:na]
	at org.springframework.data.redis.core.DefaultHashOperations.putAll(DefaultHashOperations.java:129) ~[spring-data-redis-1.7.5.RELEASE.jar:na]
	at org.springframework.data.redis.core.DefaultBoundHashOperations.putAll(DefaultBoundHashOperations.java:86) ~[spring-data-redis-1.7.5.RELEASE.jar:na]
	at org.springframework.session.data.redis.RedisOperationsSessionRepository$RedisSession.saveDelta(RedisOperationsSessionRepository.java:778) ~[spring-session-1.2.2.RELEASE.jar:na]
```

Please suggest if there is a workaround.

Cheers,
Suresh

**Comment by anshooarora** [comment]
Rdis version? MongoDB version?

**Comment by anshooarora** [comment]
Can you continue using the server with Redis?  This will be fixed in the next version.  I was able to reproduce.

**Comment by ssureshraja** [comment]
Hi @anshooarora ,
Thank you for looking into this issue.
No I cannot use the server with redis. Right now, I'm using the klov server by disabling redis.
I'm using below versions,
Redis version: 3.0.504
Mongo DB Version: 3.2.19

**Comment by anshooarora** [comment]
I meant *without.  Typo there.

**Comment by anshooarora** [comment]
This is fixed with version 0.1.0.  Please download the latest version from http://extentreports.com/community/.
